src/build/smart-contract/developers-setup.md

```diff
@@ -1,10 +1,170 @@
-# Deployment And Interaction of CW20 using cudos-noded
+---
+title: Developers Setup Version 0.1
+---
```

Comment:
could we separate this in at least three different sections please? One to deploy a full node, one for explorer/Keplr actions and another one for smart contracts